src/bezpath.rs

```diff
@@ -114,6 +115,27 @@ impl BezPath {
         BezPath::segments_of_slice(&self.0)
     }
 
+    /// Flatten the path, invoking the callback repeatedly.
```

Comment:
This should try and explain what 'flatten' means. Something like,

"Flatten the path, converting it into a `BezPath` that includes only straight lines. The `callback` argument will be called in order with each element of the generated `BezPath`.", or something.

That is: I think this documents the nitty stuff, but not the high-level stuff or how the API should be used.